spec/integration/aws_elasticsearch_domain_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'spec_helper'
+
+def policy(name)
+  <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Deny",
```

Comment:
Actually, I think the reason the user is failing to communicate is because the effect here is `Deny`.  Setting the access policy to allow all from the UI gives me a document that looks like

```json
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Principal": {
        "AWS": [
          "*"
        ]
      },
      "Action": [
        "es:*"
      ],
      "Resource": "arn:aws:es:us-east-1:#{user_id}:domain/#{name}/*"
    }
  ]
}
```